src/extension/monitor/hibernate/pom.xml

```diff
@@ -11,50 +11,71 @@
- <packaging>jar</packaging>
- <name>Monitor Hibernate Extension</name>
+	<dependencies>
+		<dependency>
+			<groupId>org.geoserver.extension</groupId>
+			<artifactId>gs-monitor-core</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-orm</artifactId>
+			<version>${spring.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>com.googlecode.genericdao</groupId>
+			<artifactId>dao</artifactId>
+			<version>1.1.0</version>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.persistence</groupId>
+					<artifactId>persistence-api</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>com.googlecode.genericdao</groupId>
+			<artifactId>search-jpa-hibernate</artifactId>
+			<version>${hibernate-generic-dao-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-entitymanager</artifactId>
+			<version>${hibernate-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-core</artifactId>
+			<version>${hibernate-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernatespatial</groupId>
+			<artifactId>hibernate-spatial</artifactId>
+			<version>${hibernate-spatial-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernatespatial</groupId>
+			<artifactId>hibernate-spatial-h2-geodb</artifactId>
+			<version>${hibernate-spatial-h2-version}</version>
+		</dependency>
 
- <dependencies>
-  <dependency>
-   <groupId>org.geoserver.extension</groupId>
-   <artifactId>gs-monitor-core</artifactId>
-   <version>${project.version}</version>
-  </dependency>
-  <dependency>
-   <groupId>javax.persistence</groupId>
-   <artifactId>persistence-api</artifactId>
-   <version>1.0</version>
-  </dependency>
-  <dependency>
-   <groupId>org.hibernate</groupId>
-   <artifactId>hibernate-entitymanager</artifactId>
-   <version>3.4.0.GA</version>
-  </dependency>
-  <dependency>
-   <groupId>org.springframework</groupId>
-   <artifactId>spring-orm</artifactId>
-   <version>${spring.version}</version>
-  </dependency>
-  <dependency>
-   <groupId>com.h2database</groupId>
-   <artifactId>h2</artifactId>
-  </dependency>
+		<dependency>
+			<groupId>org.geoserver.extension</groupId>
+			<artifactId>gs-monitor-core</artifactId>
+			<version>${project.version}</version>
+			<classifier>tests</classifier>
+			<scope>test</scope>
+		</dependency>
 
-  <dependency>
-   <groupId>org.geoserver.extension</groupId>
-   <artifactId>gs-monitor-core</artifactId>
-   <version>${project.version}</version>
-   <classifier>tests</classifier>
-   <scope>test</scope>
-  </dependency>
+	</dependencies>
 
- </dependencies>
-
- <properties>
-   <test.exclude.pattern>**/HibernateMonitorDAOTest.*</test.exclude.pattern>
- </properties>
+	<properties>
```

Comment:
This exclusion can be removed, there is a single test class and it has a different name: HibernateMonitorDAO2Test